services/ui-src/public/env-config.js

```diff
@@ -13,4 +13,6 @@ window._env_ = {
   ALLOW_DEV_LOGIN: "true",
   METRICS_USERS: "placeholder",
   IS_OFFLINE: "true",
+  GOOGLE_TAG: "placeholder",
```

Comment:
is this a leftover test? I dont see it being utilized.

Reply:
Honestly, I'm not sure if those env variables ever get used, I thought they would be for local.... so I gave them different values to see if it mattered and well, forgot to add the pretext "REACT_APP_" to that one :P 

Thanks for the catch!